rust/src/qfunction.rs

```diff
@@ -142,12 +142,10 @@ impl fmt::Display for QFunctionCore {
 ///   0
 /// };
 ///
-/// let mut qf = ceed.q_function_interior(1, Box::new(user_f));
-///
-/// qf.add_input("u", 1, EvalMode::Interp);
-/// qf.add_input("weights", 1, EvalMode::Weight);
-///
-/// qf.add_output("v", 1, EvalMode::Interp);
+/// let qf = ceed.q_function_interior(1, Box::new(user_f))
+///     .input("u", 1, EvalMode::Interp)
+///     .input("weights", 1, EvalMode::Weight)
+///     .output("v", 1, EvalMode::Interp);
```

Comment:
Note to self: We should add a check in the C code for when we add a Operator field that the basis dim and ncomp match the QFunction field size and eval mode provided. I've generated segfaults in our C (and Rust!) examples due to a mismatch. (separate PR)